src/admin/python/lsst/qserv/admin/itest_load.py

```diff
@@ -121,7 +126,7 @@ def _create_ref_db(ref_db_admin: str, name: str) -> None:
                 execute(cursor, stmt)
 
 
-def _create_ref_table(cnx: mysql.connector.connection, db: str, schema_file: str) -> None:
+def _create_ref_table(cnx: MySQLConnectionAbstract, db: str, schema_file: str) -> None:
```

Comment:
Maybe also change docstring for `cnx` parameter below to specify the same new type?

Reply:
Yes, thanks for the catch...